api/edge_api/identities/export.py

```diff
@@ -0,0 +1,181 @@
+    return {
+        "model": "traits.trait",
+        "fields": {
+            "identity": [identifier, environment_api_key],
+            "created_date": timezone.now().isoformat(),
+            "trait_key": trait["trait_key"],
+            **trait_value_data,
+        },
+    }
+
+
+def export_edge_identity(
+    identifier: str, environment_api_key: str, created_date: str
+) -> dict:
+    return {
+        "model": "identities.identity",
+        "fields": {
+            "identifier": identifier,
+            "created_date": created_date,
+            "environment": [environment_api_key],
+        },
+    }
+
+
+def export_edge_feature_state(
+    identifier: str,
+    environment_api_key: str,
+    featurestate_uuid: str,
+    feature_id: int,
+    enabled: bool,
+) -> dict:
+    global feature_id_to_uuid
+    if feature_id not in feature_id_to_uuid:
+        feature_id_to_uuid[feature_id] = _get_feature_uuid(feature_id)
+    feature_uuid = feature_id_to_uuid[feature_id]
+
+    return {
+        "model": "features.featurestate",
+        "fields": {
+            "uuid": featurestate_uuid,
+            "created_at": timezone.now().isoformat(),
+            "updated_at": timezone.now().isoformat(),
+            "live_from": timezone.now().isoformat(),
+            "feature": [feature_uuid],
+            "environment": [environment_api_key],
+            "identity": [
+                identifier,
+                environment_api_key,
+            ],
+            "feature_segment": None,
+            "enabled": enabled,
+            "version": 1,
+        },
+    }
+
+
+def export_featurestate_value(
+    featurestate_value: typing.Any, featurestate_uuid: str
+) -> dict:
+    if isinstance(featurestate_value, Decimal):
+        if featurestate_value.as_tuple().exponent == 0:
+            featurestate_value = int(featurestate_value)
+
+    fsv_data = FeatureState().generate_feature_state_value_data(featurestate_value)
+    fsv_data.pop("feature_state")
+
+    return {
+        "model": "features.featurestatevalue",
+        "fields": {
+            "uuid": uuid.uuid4(),
+            "feature_state": [featurestate_uuid],
+            **fsv_data,
+        },
+    }
+
+
+def export_mv_featurestate_value(
+    featurestate_uuid: str, mv_feature_option_id: int
+) -> dict:
+
+    global mv_feature_option_id_to_uuid
+    if mv_feature_option_id not in mv_feature_option_id_to_uuid:
+        mv_feature_option_id_to_uuid[mv_feature_option_id] = (
+            _get_mv_feature_option_uuid(mv_feature_option_id)
+        )
+    mv_feature_option_uuid = mv_feature_option_id_to_uuid[mv_feature_option_id]
+
+    return {
+        "model": "multivariate.multivariatefeaturestatevalue",
+        "fields": {
+            "uuid": uuid.uuid4(),
+            "feature_state": [featurestate_uuid],
+            "multivariate_feature_option": [mv_feature_option_uuid],
+            "percentage_allocation": 100.0,
```

Comment:
Why is this hard coded to `100.0`?

Reply:
It's because an identity override can only be 100 percent(for mv)

Reply:
As per my comment above, I'm not sure this is strictly true.

Reply:
Done, it is not hardcoded anymore